crates/solidity/inputs/schema/grammar/topics/01-file-structure/08-punctuation.yml

```diff
@@ -0,0 +1,79 @@
+- name: "OpenParen"
```

Comment:
There is some overlap between some tokens in punctuation vs operators. For example, for `=`, we have two tokens:

* `AssignmentOperator` under `operators.yml`, which is used in things like `AssignmentExpression`.
* `Equal` under `punctuation.yml`, which is used in things like `VariableDeclarationStatement`.

Both have the same content and meaning. But having both is more work/overhead for the user, and is prone to divergance and suble bugs in the future. I think I already see one, as `Equal` doesn't have the correct `trailingContext` as `AssignmentOperator`.

I suggest reusing the ones in operators everywhere, and removing the copies here.

Reply:
done